src/test/java/org/jenkins_ci/plugins/flexible_publish/ConfigurationTest.java

```diff
@@ -30,6 +30,7 @@
 import java.util.Collections;
 import java.util.List;
 
+import com.gargoylesoftware.htmlunit.WebClient;
```

Comment:
I recommend `org.jvnet.hudson.test.JenkinsRule.WebClient` instead as it provides Jenkins-specific utility methods.